clinics/cvs.py

```diff
@@ -29,7 +29,7 @@ def get_locations(self):
             },
             "requestPayloadData": {
                 "selectedImmunization": ["CVD"],
-                "distanceInMiles": 35,
+                "distanceInMiles": int(os.environ["RADIUS"]),
```

Comment:
I missed this one, thanks for the catch

Reply:
It's getting serialized to json, so we don't need to cast as int here
```suggestion
                "distanceInMiles": os.environ["RADIUS"],
```